aether-kernel/conf/pip/requirements.txt

```diff
@@ -31,17 +34,26 @@ django-prometheus==1.0.15
 django-rest-swagger==2.1.2
 django-reversion==3.0.0
 django-reversion-compare==0.8.4
+django-storages==1.6.6
 django-ums-client==0.2.6
 djangorestframework==3.8.2
 djangorestframework-csv==2.1.0
 djangorestframework-filters==0.10.2
+docutils==0.14
 drf-dynamic-fields==0.3.0
 drf-openapi==1.3.0
 flake8==3.5.0
 flake8-quotes==1.0.0
+google-api-core==1.3.0
+google-auth==1.5.0
+google-cloud-core==0.28.1
+google-cloud-storage==1.10.0
+google-resumable-media==0.3.1
+googleapis-common-protos==1.5.3
```

Comment:
@lordmallam did you introduce these dependencies manually? They disappear when you try to freeze pip dependencies :disappointed: